[2024-09-27T14:13:10.282+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-27T14:13:10.305+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: find_activity.get_activity manual__2024-09-27T14:12:55.815003+00:00 [queued]>
[2024-09-27T14:13:10.316+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: find_activity.get_activity manual__2024-09-27T14:12:55.815003+00:00 [queued]>
[2024-09-27T14:13:10.318+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-27T14:13:10.341+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): get_activity> on 2024-09-27 14:12:55.815003+00:00
[2024-09-27T14:13:10.347+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=71) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-27T14:13:10.349+0000] {standard_task_runner.py:72} INFO - Started process 72 to run task
[2024-09-27T14:13:10.349+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'find_activity', 'get_activity', 'manual__2024-09-27T14:12:55.815003+00:00', '--job-id', '62', '--raw', '--subdir', 'DAGS_FOLDER/find_activity.py', '--cfg-path', '/tmp/tmpv3fad462']
[2024-09-27T14:13:10.351+0000] {standard_task_runner.py:105} INFO - Job 62: Subtask get_activity
[2024-09-27T14:13:10.413+0000] {task_command.py:467} INFO - Running <TaskInstance: find_activity.get_activity manual__2024-09-27T14:12:55.815003+00:00 [running]> on host 0060ec602991
[2024-09-27T14:13:10.535+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='find_activity' AIRFLOW_CTX_TASK_ID='get_activity' AIRFLOW_CTX_EXECUTION_DATE='2024-09-27T14:12:55.815003+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-27T14:12:55.815003+00:00'
[2024-09-27T14:13:10.537+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-27T14:13:12.286+0000] {python.py:240} INFO - Done. Returned value was: {'activity': 'Make a new friend', 'availability': 0, 'type': 'social', 'participants': 1, 'price': 0, 'accessibility': 'Few to no challenges', 'duration': 'minutes', 'kidFriendly': True, 'link': '', 'key': '1000000'}
[2024-09-27T14:13:12.312+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-27T14:13:12.313+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=find_activity, task_id=get_activity, run_id=manual__2024-09-27T14:12:55.815003+00:00, execution_date=20240927T141255, start_date=20240927T141310, end_date=20240927T141312
[2024-09-27T14:13:12.376+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-27T14:13:12.403+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-27T14:13:12.409+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
